Marlin/src/lcd/extui_malyan_lcd.cpp

```diff
@@ -474,6 +474,29 @@ namespace ExtUI {
     write_to_lcd_P("}");
   }
 
+  #if HAS_PID_HEATING
+
+    void onPidTuning(const result_t rst) {
+      // Called for temperature PID tuning result
+      //SERIAL_ECHOLNPAIR("OnPidTuning:", rst);
+      switch (rst) {
+        case PID_BAD_EXTRUDER_NUM:
+          write_to_lcd_P(GET_TEXT(MSG_PID_BAD_EXTRUDER_NUM));
```

Comment:
This won't actually show anything. Hang on and I'll pull this and test. This LCD is extremely limited in what it can show, so you either need to dispatch it as an error or not show it.

Reply:
I've now added the `{E: }` wrapper. Looks like it should work.